azure_pipelines/verify_msalcpp_per_pr_ios.yml

```diff
@@ -40,17 +40,16 @@ jobs:
   - task: UsePythonVersion@0
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->